### ISSUE TITLE
Should Line/Polygon/Path points be absolute or relative to position ?

### ISSUE THREAD

**Issue by GMartigny** [GMartigny/pencil.js#18]
Points of a Line object have absolute position. That way, in order to draw a polygon between 3 shapes, it's easy to write:

```js
const line = new Polugon([
	shape1.position,
	shape2.position,
	shape3.position,
]);
```
This means that moving one position don't change the others.

In the other hand, relative position could allow to move the polygon position and keep the polygon shape.

**Comment by GMartigny** [comment]
Relatives points allow shapes to be easily draggable by only modifying shape's position.

**Comment by GMartigny** [comment]
Fixed by f750ca8
